### PR TITLE
adding build instructions

### PR DESCRIPTION
**Pull request by rajatshuvro**

### CODE REVIEW

File: docs/data-sources/clinvar.mdx [hunk 1]
```diff
@@ -297,14 +297,98 @@ May have multiple significances listed.
 :::caution Known Issues
 * The XML file contains ~1k more entries (out of 162K) than the VCF file
 * The XML file does not have a field indicating that a record is associated with the reference base - something that was present in VCF
-* The XML file contains entries (e.g. RCV000016645 version=1) which have IUPAC ambiguous bases ("R", "Y", "H", 
+* The XML file contains entries (e.g. RCV000016645 version=1) which have IUPAC ambiguous bases ("R", "Y", "H",
 etc.) as their alternate allele
 :::
 
 ## Download URL
```
Comment (MichaelStromberg) [inline docs/data-sources/clinvar.mdx]: Should be Download URLs

File: docs/data-sources/clinvar.mdx [hunk 2]
```diff
@@ -297,14 +297,98 @@ May have multiple significances listed.
...
 [ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz](ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz)
 
+[https://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/clinvar_variation/ClinVarVariationRelease_00-latest.xml.gz](https://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/clinvar_variation/ClinVarVariationRelease_00-latest.xml.gz)
+
 ## JSON Output
 
 <JSON />
+
+## Building the supplementary files
+The ClinVar ``.nsa`` for Nirvana can be build using the ``SAUtils`` command's ``clinvar`` subcommand.
```
Comment (MichaelStromberg) [inline docs/data-sources/clinvar.mdx]: can be built

File: docs/data-sources/clinvar.mdx [hunk 3]
```diff
@@ -297,14 +297,98 @@ May have multiple significances listed.
...
+### Source data files
+Two input ``.xml`` files and a ``.version`` file is required in order to build the ``.nsa`` file. Here is a set of files.
```
Comment (MichaelStromberg) [inline docs/data-sources/clinvar.mdx]: are required
You should have the following files:

**Review by MichaelStromberg — CHANGES_REQUESTED** [review]

File: docs/data-sources/clinvar.mdx (hunk 3 above)
Comment (rajatshuvro) [reply docs/data-sources/clinvar.mdx]: updated accordingly

**Pull request merged**